.gitignore

```diff
@@ -59,6 +59,7 @@ $RECYCLE.BIN/
 *.swm
 *.out
 *.pid
+package-lock.json
```

Comment:
ao invés de vocês ignorarem o package-lock, o ideal seria vcs sempre manterem ele atualizado, e ao fazer pull da branch, instalar os pacotes usando `npm ci` ao invés de `npm i`

ref: https://vidafullstack.com.br/sem-categoria/atualizacao-do-arquivo-package-lock-json/

Reply:
isso também se aplica para o yarn?

Reply:
Sim sim. Dá uma lida: 
https://chore-update--yarnpkg.netlify.app/pt-BR/docs/yarn-lock
https://stackoverflow.com/questions/44552348/should-i-commit-yarn-lock-and-package-lock-json-files
(em inglês)